catalog/core/catalog-core-validationparser/src/main/java/ddf/catalog/validation/impl/ValidationParser.java

```diff
@@ -221,11 +229,32 @@ private void parseValidators(Map<String, Object> root, Outer outer) {
 
         Map<String, List<Outer.Validator>> validators = new HashMap<>();
         for (Map.Entry<String, Object> entry : ((Map<String, Object>) root.get(VALIDATORS_PROPERTY)).entrySet()) {
-            String rejson = JsonFactory.create()
-                    .toJson(entry.getValue());
-            List<Outer.Validator> lv = JsonFactory.create()
-                    .readValue(rejson, List.class, Outer.Validator.class);
-            validators.put(entry.getKey(), lv);
+            if (entry.getValue() instanceof ValueList) {
+                ValueList argumentList = (ValueList) entry.getValue();
+                List<Outer.Validator> validatorList = new ArrayList<>();
+                for (Object object : argumentList) {
+                    if (object instanceof LazyValueMap) {
+                        LazyValueMap lazyValueMap = (LazyValueMap) object;
+                        final String comp = (String) lazyValueMap.get(VALIDATOR_PROPERTY);
+                        String rejson = JsonFactory.create()
```

Comment:
This was an existing var name; i think it was named `rejson` to convey that the object was converted back to JSON for this method. Renamed it to `json` as I'm not sure how important this detail is.